README.md

```diff
@@ -61,6 +61,13 @@ pmpc_layer = PMPCLayer(chain_length=20, pattern=pattern, tile_x=3, tile_y=2)
 
 ![Zwitterionic brushes on beta-cristobalite substrate](docs/images/pmpc.png)
 
```

Comment:
Might be great to have this as a new section, I kind of skipped over it when skimming.
```suggestion
#### User Recipes
```


Or something to that effect.